include/rdma/fi_eq.h

```diff
@@ -75,10 +77,18 @@ struct fid_wait {
 	struct fi_ops_wait	*ops;
 };
 
+#ifndef _WIN32
 struct fi_mutex_cond {
 	pthread_mutex_t		*mutex;
 	pthread_cond_t		*cond;
 };
+#else /* _WIN32 */
+struct fi_critsec_cond {
+	CRITICAL_SECTION	*critsec;
+	CONDITION_VARIABLE	*cond;
+};
+#endif /* _WIN32 */
```

Comment:
not sure I got your idea...
which level of datatype you want to cover by macro?
```
#define FI_WAIT_CRITSEC_COND fi_mutex_cond
```
or something else?